### ISSUE TITLE
FormsPlot control not available in the Toolbox

### ISSUE THREAD

**Issue by wellsw** [ScottPlot/ScottPlot#2900]
## Bug Report

The FormsPlot control does not appear in the Toolbox.
I tried creating projects with both .NET Framework 4.6.2 and 4.7.2 and the FormsPlot still did appear in the Toolbox after installing ScottPlot via NuGet.
I even tried creating projects targeting .NET 6.0 and .NET 7.0, after installing ScottPlot via NuGet, the FormsPlot control did not appear in the Toolbox.
In all cases I tried rebuilding the project, and restarting Visual Studio and the FormsPlot control still did not appear in the Toolbox.

System Details
ScottPlot Version: 4.1.67
Operating System: Windows 11 Pro, 64 bit
Visual Studio Version: 17.7.2
NuGet PackageManager Version: 6.7.0
Application Type: WinForms
.NET Version: Tried both .NET 7.0 and .NET 6.0, .Net Framework 4.7.2, .Net Framework 4.6.2

**Comment by wsw168email** [comment]
Same question.
System Details
ScottPlot Version: 4.1.68
Operating System: Windows 11 Pro, 64 bit
Visual Studio Version: 17.7.5
NuGet PackageManager Version: 6.7.0
Application Type: WinForms
.NET Version: Tried both .NET 7.0 and .NET 6.0, .Net Framework 4.7.2, .Net Framework 4.6.2

**Comment by Minnator** [comment]
Same question.
Sysetm Details:
ScottPlot Version: 4.1.68
Operating System: Windows 10 Pro, 64 bit
Visual Studio Version: 17.7.5
NuGet PackageManager Version: 6.7.0
Application Type: WinForms
.NET Version: Tried both .NET 7.0 and .NET 6.0, .Net Framework 4.7.2, .Net Framework 4.6.2

**Comment by swharden** [comment]
Hi @wellsw, @wsw168email, and @Minnator

Unfortunately I am unable reproduce this issue so I'm struggling to offer recommendations for how to fix it! Perhaps confirm the steps you are following are the same as the steps I am using to test this on my computer?

* I'm using the latest version of Visual Studio Community (free)

* I created a new .NET Framework 4.6.2 app

* I added the ScottPlot.WinForms NuGet package (not just the ScottPlot package, and not a prerelease package)

* The FormsPlot control appears in the toolbox for me right away

![image](https://github.com/ScottPlot/ScottPlot/assets/4165489/f379c73b-2f27-4b9a-b499-48bb1ba5028d)

![image](https://github.com/ScottPlot/ScottPlot/assets/4165489/b012729f-3e32-477c-a6c9-182874e43bb2)

**Comment by swharden** [comment]
In extreme cases (using very old versions of Visual Studio) if the FormsPlot item never appears in the toolbox, you can drop a `Panel` where you want a plot to be and add the FormsPlot control programmatically. This strategy is described here: https://scottplot.net/faq/add-winforms-programmatically/

**Comment by Minnator** [comment]
![grafik](https://github.com/ScottPlot/ScottPlot/assets/50293050/64ec0606-1a14-46b3-89df-7f4e8cb67420)
![grafik](https://github.com/ScottPlot/ScottPlot/assets/50293050/d4e5b205-c4ae-4697-b92b-7dbca6a1040e)
![grafik](https://github.com/ScottPlot/ScottPlot/assets/50293050/0246cd30-9274-4423-8b62-0d35be477732)

Hi,
If I Mirror ur steps it does not work for me, ik that it is possible to create them programically but using the designer would be amazing.
- I'm using the latest version of Visual Studio Community (free)
- I created a new .NET Framework 4.6.2 app
- I added the ScottPlot.WinForms NuGet package (not just the ScottPlot package, and not a prerelease package)

**Comment by swharden** [comment]
Hi @Minnator, thanks for getting back to me with screenshots!

Could this be your issue? Create a new project and try again but make sure this box is unchecked

![image](https://github.com/ScottPlot/ScottPlot/assets/4165489/c6dc3b7a-7b98-4f9b-a262-cec1ad326101)

**Comment by Minnator** [comment]
Yes, now it shows for me. Thx for the patience and help

**Comment by hanshantong** [comment]
The same problem for me. My environment is window 11 64 bit, Visual studio 2022 (community), ScottPlot version 5.0.25. The "OpenTK" and "SkiaSharp.Views" apeared in the toolbot but "ScottPlot.WinForms" does not. 
I also try old version of Visudl studio (e.g. vs2017, vs2019), however, the same problem occured. And I also install old version of ScottPlot, the "ScottPlot.WinForms" apeared in some cases, But if you add a FormsPlot to Winform app, it say "The FormsPlot can't be added in the toolbox", that's very weird.

**Comment by fisoul** [comment]
Same problem for me.
ScottPlot / ScottPlot.Winforms: 5.0.34
Windows 11 64 bit famliy version
Visual Studio 2022 Community
Only OpenTK and SkiaSharp shows up.

**Comment by swharden** [comment]
Thanks for reporting this @fisoul! The workaround seems to be the strategy described in the link below (place a `Panel` where you want the plot, then create a `FormsPlot` programmatically and add it to the `Panel`)

https://scottplot.net/faq/add-winforms-programmatically/

**Comment by GroovyGooRoo** [comment]
For what it's worth I had a similar problem with the following sequence and solution.

The FormPlot1 control was added to a Form (frmChart) with the name plt1.  As expected in the frmChart.Designer.cs 'InitialiseComponent' function there was the line plt1 = new ScottPlot.WinForms.FormsPlot();
frmChart 'View Designer' was working well and then (after a Windows update I think though I'm not certain) the Designer would not open by clicking 'View Designer'. The error message was 'error CS0103: The name 'plt1_Load' does not exist in the current context '

When I checked under the plt1 listing in frmChart.Designer.cs there was a line 'plt1.Load += plt1_Load;'. How it came to be there I don't know. I commented this line out and saved the file. The 'View Designer' then worked.
